### PR TITLE
Updated ambiguous text

### PR DESCRIPTION
**Pull request by StephenOTT**
Fixes issue from: https://github.com/orocrm/platform/issues/388

### CODE REVIEW

File: user_guide/integrations_overview.rst [hunk 1]
```diff
@@ -5,8 +5,7 @@ OroCRM Integrations Overview
 
 OroCRM supports two types of integrations: 
 
-- Integrations applied to the whole system. There can only be one integration with a third-party system for an OroCRM 
-  instance. These integrations can be enabled/disabled and defined in the *"System → Configuration → Integration"*.
+- Integrations are applied system wide. An instance of an integration, such as with a third-party system, can only be applied once per OroCRM instance. These integrations can be enabled/disabled and defined in the *"System → Configuration → Integration"*.
```
Comment (sergeyz) [inline user_guide/integrations_overview.rst]: This limitation affects developers only. User can create multiple connections to any 3rd party system without any limits. I propose to omit limit related part in user guide

Comment (StephenOTT) [inline user_guide/integrations_overview.rst]: @sergeyz are you suggesting to remove `such as with a third-party system` ?

How can a user create multiple connections to a 3rd part system using the same integration?  (Understanding that you duplicate the bundles... but based previous explanations, an Integration can only be "activated" once per orocrm instance.  This correct?

I use the correct out of the box / demo UI as the example:
screen shot 2016-02-01 at 2 08 18 pm

The user cannot add multiple google single sign on credentials.

**Comment by orocla** [comment]
Thank you for your pull request.

It looks like this may be your first contribution to an Oro, Inc. open source project, in which case you'll need to sign a Contributor License Agreement (CLA) at http://www.orocrm.com/contributor-license-agreement/

If you've already signed the CLA, it's possible we don't have your GitHub username or you're using a different email address. GitHub uses the email address you set in your local Git configuration to associate commits with your GitHub account. Please sign the CLA again using the correct GitHub username and email address or see this help article on [setting the email on your git commits](https://help.github.com/articles/setting-your-email-in-git/).

Once you've signed the CLA, please allow for some time for the submission to be processed.

**Comment by mbessolov** [comment]
That entire page on integrations needed a little bit of rework - see #196

These are the relevant bits from #196 on this particular topic:

> • Integrations that are configured at the system level and share the same configuration for all [organizations](http://www.orocrm.com/documentation/index/current/user-guide/glossary/#term-29)  in an OroCRM instance. These integrations (or any combination of them) can be enabled/disabled and configured in the "System → Configuration → Integration".
> ...
> ...
> • Integrations that are configured at the [organization](http://www.orocrm.com/documentation/index/current/user-guide/glossary/#term-29) or [channel](http://www.orocrm.com/documentation/index/current/glossary/#term-channel) level. These integrations allow for adjustment of third-party system integration parameters (e.g. credentials, account IDs, synchronizations settings, etc.) for different channels and/or organizations.

I hope it provides enough context and highlights the difference between the system-wide integrations and integrations that are configured at the organization/channel level.

Please let me know if it you think it needs more adjustments.

For us, the developers of these products, it is sometimes hard to assess the quality of the documentation because it often feels like a lot of repetition on so familiar topics. We highly appreciate the community feedback on our products' documentation and welcome any suggestions on how to make it better.

Thank you,
Michael

**Comment by StephenOTT** [comment]
@mbessolov your updated text is very clear... Makes sense now!

Totally understand the repetitiveness.  

I was looking at the integrations from the OroPlatform/BAP level, so in terms of your comments about the Channel integrations, is this just how you implemented the Channels bundle?

**Comment by mbessolov** [comment]
The integrations with Magento, Zendesk, Mailchimp and Dotmailer allow that level of configuration because a **channel** has its specific purpose in the context of a CRM application.

It is either a way of how the business is interacting with its existing and potential customers (an online store, a brick-and-mortar store, a call-center, an inbound marketing campaign) or simply a way how the data is retrieved from a third-party system. Or any combination of those.

For example some businesses treat all their online sales as a single "online sales" channel, others believe there should be separate "Store X", "Store Y", "Store Z" channels for each of the online properties they operate. The real power of OroCRM is in its flexibility and ability to support all these scenarios and aggregate the data the way the customer wants.

If you are implementing an integration with a 3-rd party system it's up to you as a developer to decide whether your customers will need that level of separation and ability to treat data differently per channel or not. If it has anything to do with sales, marketing or support - than probably it could be more beneficial to a customer to have that channel separation than not. It may simplify integrating multiple systems as usually sales/marketing/support software has one or another notion of customer interaction channels. In this case it could also be beneficial (this time to a systems integrator) to use OroCRM (and not OroPlatform) as a starting point.

If there is nothing to do with sales/marketing/support or any customer interaction, then there is probably no need to have a notion of channels in the system. In such a case the developer has a simpler choice to make - whether or not his integration needs to be configured independently per organization, or it is to be setup once on the system level.

Or they may need to introduce a new, quite different level of separation. In an imaginable scenario of implementing a content management system I might need data separated per language/locale, so that for the content in every language I may use a separate set of 3-rd party content providers/document management systems and repositories.

**Comment by mbessolov** [comment]
Closing this PR in favor of #196 which has this and other fixes.

**Comment by StephenOTT** [comment]
:+1: @mbessolov thanks for the very detailed reply!

**Pull request closed without merging**